### PR TITLE
Improve docs

### PR DESCRIPTION
**Pull request by cr3wdayt5p**
Improve docs:
 - Getting started:
   - Styling of install commands.
   - Remove references to the removed `--recurse` flag.
 - JetBrains rider:
   - Styling of install command.
   - Update screenshot.
   - Simplify info on Rider 2022.3 (and later) now respecting the default settings of fantomas (not really relevant to new users of F#/Fantomas). Also no need to have to keep this page updated will all current default settings.

### CODE REVIEW

**Review by dawedawe — APPROVED** [review]
Great improvements @cr3wdayt5p . Thanks a lot!

File: docs/docs/end-users/Rider.md [hunk 1]
```diff
@@ -6,56 +6,16 @@ index: 7
...
-fsharp_blank_lines_around_nested_multiline_expressions=true 
-fsharp_bar_before_discriminated_union_declaration=false
-fsharp_keep_max_number_of_blank_lines=100
-fsharp_strict_mode=false
+dotnet tool install fantomas
 ```
 
+<img class="mt-2" src="{{root}}/images/rider-fantomas.png" alt="drawing" width="70%"/>
+
+Prior to Rider 2022.3 it did not [respect](https://youtrack.jetbrains.com/issue/RIDER-83997/Rider-doesnt-respect-Fantomas-default-settings-not-explicitly-set-in-editorconfig) the default settings of Fantomas.
```
Comment (auduchinok) [inline docs/docs/end-users/Rider.md:18]: Just a fun story to share: at some point Rider took the actual defaults from Fantomas and hardcoded them to its settings. Later Fantomas has changed the defaults, so the ones in Rider have become outdated, and for some time we were unfortunately stuck with them. It was never an intention to not respect the settings, just a coincidence of how the things have evolved here 🙂

Comment (cr3wdayt5p) [reply docs/docs/end-users/Rider.md:18]: I think that a lot of software evolve like that :) A "good enough" solution is often the best starting point, and then when/if it breaks it is easier to design the "good" solution (more knowledge available).

But Rider sure does have some fun formatting quirks :) It is pretty confusing to figure out what settings it applies and when. E.g. when I edit a `.fsproj` file it sort of uses a mix of the "code style" settings for XML-files (specified in the Settings UI), and the settings I have specified in `.editorconfig` – but this is not shown anywhere in settings UI (i.e. which settings are overriden by `.editorconfig` – or the way around). And then when Rider modifies `.fsproj` it uses a whole different set of formatting rules :)

Comment (auduchinok) [reply docs/docs/end-users/Rider.md:18]: Oh, the `.fsproj` story is also interesting. When editing the file manually then IntelliJ rules are used, but when Rider modifies it from code it actually delegates a lot of the work to MSBuild and it doesn't seem to follow what the rest of the IDE does. I'm not sure how easy/feasible it's to intercept the MSBuild modifications, but it seems it was difficult enough to become less prioritized.

**Pull request merged**